libs/@guardian/core-web-vitals/src/index.test.ts

```diff
@@ -1,5 +1,5 @@
+import * as libs from '@guardian/libs';
```

Comment:
I think this is the wrong import.

```suggestion
import { logger } from '@guardian/libs';
```